README.md

```diff
@@ -23,10 +23,13 @@ Proxy-Wasm extensions across different proxies.
 
 * [AssemblyScript]
 
+* [TinyGo]
```

Comment:
While you're here, could you remove the empty lines between all the SDKs? Not sure why I added them.

Reply:
give me a sec

Reply:
done